install/installer/cluster_dev.py

```diff
@@ -256,23 +388,26 @@ def parse_cli_args():
     if cli.git_user_name:
         UserNameValidator.validate(cli.git_user_name, interactive=False)
 
+    if cli.cloud and cli.cloud_provider:
+        if cli.cloud_provider not in CLOUD_PROVIDERS[cli.cloud]:
+            _exit_(
+                f'Cloud provider can be: {CLOUD_PROVIDERS[cli.cloud_provider]}, ' +
+                f'but provided: {cli.cloud_provider}',
+            )
+
```

Comment:
That's a lot of unnecessary indentation:
```suggestion
    if (
        cli.cloud and cli.cloud_provider and
        cli.cloud_provider not in CLOUD_PROVIDERS[cli.cloud]
    ):
        _exit_(
            f'Cloud provider can be: {CLOUD_PROVIDERS[cli.cloud_provider]}, ' +
            f'but provided: {cli.cloud_provider}',
        )

```

Reply:
```python3
    if (
        cli.cloud
        and cli.cloud_provider
        and cli.cloud_provider not in CLOUD_PROVIDERS[cli.cloud]
    ):
```

raises https://wemake-python-stylegui.de/en/latest/pages/usage/violations/consistency.html#wemake_python_styleguide.violations.consistency.MultilineConditionsViolation